src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx

```diff
@@ -152,9 +158,18 @@ class OrganizationTeamsProjectsView extends React.Component {
         <SettingsPageHeader title={t('Projects by Team')} />
         <div className="col-md-9">
           <div className="team-list">
-            {Object.keys(projectsByTeam).map(teamSlug => {
-              return this.renderTeamNode(teamSlug, projectsByTeam[teamSlug]);
-            })}
+            {Object.keys(projectsByTeam)
+              .sort()
```

Comment:
https://github.com/getsentry/sentry/blob/feat/master/src/sentry/static/sentry/app/utils.jsx#L151

would this help? (not blocking)

Reply:
that link is 404ing, but the default [array sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) sorts by unicode code points, which i think is basically what we'd want